.github/workflows/cache-issues.yaml

```diff
@@ -4,6 +4,10 @@ on:
     - cron: '0 */6 * * *'
   workflow_dispatch:
 
+env:
+  CLOUDIUS_SYSTEMS_CANONICAL_ID: 0c2716c5c3e86717000f7af99a8c82c5f776b86179f73d4985de88ceff86a9a4
```

Comment:
Lets put those into secrets

Reply:
Don't have permission to manager secrets here.
Could you please help with that?

Reply:
I've created secrets with those names